SConstruct

```diff
@@ -588,6 +588,10 @@ else:
 				"/wd4003",  # suppress warning "not enough arguments for function-like macro invocation 'BOOST_PP_SEQ_DETAIL_IS_NOT_EMPTY'". Needed for USD.
 				"/wd4702",  # suppress warning "unreachable code". Need for OpenVDB.
 				"/wd4180",  # suppress warning "qualifier applied to function type has no meaning; ignored". Needed for OpenVDB
+				"/wd4146",  # suppress warning "unary minus operator applied to unsigned type, result still unsigned" (from Cryptomatte::MurmurHash3_x86_32())
+				"/D_CRT_NONSTDC_NO_WARNINGS",  # suppress warnings about deprecated POSIX names. The names are deprecated, not the functions, so this is safe.
+				"/D_SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING",
+				"/D_SILENCE_CXX17_SHARED_PTR_UNIQUE_DEPRECATION_WARNING",
```

Comment:
I'm curious about these - are they something we could fix in the source instead?